libs/blocks/accordion/accordion.js

```diff
@@ -55,11 +40,7 @@ function createItem(accordion, id, heading, num) {
   const dt = createTag('dt', { role: 'heading', 'aria-level': 3 }, button);
   const dd = createTag('dd', { role: 'region', 'aria-labelledby': triggerId, id: panelId, hidden: true }, panel);
 
-  button.addEventListener('click', (e) => { handleClick(e.target, dt, dd); });
-  dt.addEventListener('focusin', () => { handleFocus(dt, dd); });
```

Comment:
Accessibility is lost with this - tab, spacebar etc to navigate between accordion items. Is this going to be handled another way?

Reply:
No plans to handle it another way. We can keep these in. 
It does looks to be working natively via aria labels with out these focus events accessibility wise. (being able to tab-focus, tab-key, and space/return navigate). Maybe there's value to having them i'm not privy too. 
@fullcolorcoder lmk if you think we should keep em or any thoughts thanks.

Reply:
@ryanmparrish Did some testing on it, looks great thanks.